[2025-07-31T12:15:31.768+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-31T12:15:31.784+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: world_population_ETL_dag_43.create_table_task manual__2025-07-31T12:15:23.598722+00:00 [queued]>
[2025-07-31T12:15:31.791+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: world_population_ETL_dag_43.create_table_task manual__2025-07-31T12:15:23.598722+00:00 [queued]>
[2025-07-31T12:15:31.792+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-07-31T12:15:31.802+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): create_table_task> on 2025-07-31 12:15:23.598722+00:00
[2025-07-31T12:15:31.814+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'world_population_ETL_dag_43', 'create_table_task', 'manual__2025-07-31T12:15:23.598722+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/project_01_dag.py', '--cfg-path', '/tmp/tmp14tyg9c2']
[2025-07-31T12:15:31.816+0000] {standard_task_runner.py:105} INFO - Job 20: Subtask create_table_task
[2025-07-31T12:15:31.816+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=383) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-31T12:15:31.817+0000] {standard_task_runner.py:72} INFO - Started process 385 to run task
[2025-07-31T12:15:31.853+0000] {task_command.py:467} INFO - Running <TaskInstance: world_population_ETL_dag_43.create_table_task manual__2025-07-31T12:15:23.598722+00:00 [running]> on host 1e069286f646
[2025-07-31T12:15:31.918+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='givi-abe' AIRFLOW_CTX_DAG_ID='world_population_ETL_dag_43' AIRFLOW_CTX_TASK_ID='create_table_task' AIRFLOW_CTX_EXECUTION_DATE='2025-07-31T12:15:23.598722+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-31T12:15:23.598722+00:00'
[2025-07-31T12:15:31.919+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-31T12:15:31.933+0000] {logging_mixin.py:190} INFO - ✅ Connected to PostgreSQL
[2025-07-31T12:15:31.934+0000] {logging_mixin.py:190} INFO - ✅ Table created or already exists.
[2025-07-31T12:15:31.934+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-31T12:15:31.943+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-31T12:15:31.943+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=world_population_ETL_dag_43, task_id=create_table_task, run_id=manual__2025-07-31T12:15:23.598722+00:00, execution_date=20250731T121523, start_date=20250731T121531, end_date=20250731T121531
[2025-07-31T12:15:31.990+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-31T12:15:32.022+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-31T12:15:32.024+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
